modal/image.py

```diff
@@ -314,7 +314,7 @@ def extend(self, **kwargs) -> "_Image":
         return _Image._from_args(base_images={"base": self}, **kwargs)
 
     @typechecked
-    def copy(self, mount: _Mount, remote_path: Union[str, Path] = ".") -> "_Image":
+    def add_mount(self, mount: _Mount, remote_path: Union[str, Path] = ".") -> "_Image":
```

Comment:
I think `os.PathLike` might be better than `Path` for what we accept here, since it's broader. Probably for a separate PR, since it was already this way.